samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/PublishAssessmentListener.java

```diff
@@ -324,6 +325,12 @@ private void publish(AssessmentFacade assessment,
                                                  "gradebook_exception_title_invalid");
         FacesContext.getCurrentInstance().addMessage(null, new FacesMessage(err));
         throw new AbortProcessingException(gbe);
+    } catch (InvalidCategoryException gbe) {
+		log.warn(gbe.getMessage());
```

Comment:
More informative log message?
```suggestion
		log.warn("Incorrect gradebook category settings: {}", gbe.toString());
```